source/index.html.md

```diff
@@ -156,6 +156,7 @@ error categories, the last two digits define specific errors.
 * 0359: The password is too long
 * 0362: Invalid callback URL
 * 0363: Invalid Transaction ID (Either non-existent or does not belong to the user)
+* 0364: Password doesn't meet security requirements
```

Comment:
Do we publish these security requirements somewhere so that users can refer to them?

Reply:
No, but the error message in the API states what is wrong.